src/ucp/api/ucp_compat.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) Mellanox Technologies Ltd. 2001-2017.  ALL RIGHTS RESERVED.
+ *
+ * See file LICENSE for terms.
+ */
+
+
+#ifndef UCP_COMPAT_H_
+#define UCP_COMPAT_H_
+
+
+#include <ucp/api/ucp_def.h>
+
+
+/**
+ * @ingroup UCP_COMM
+ * @deprecated Replaced by @ref ucp_request_test.
+ */
+int ucp_request_is_completed(void *request);
```

Comment:
What happens with the applications that already use the API ?
Shell we bump up the version ?